### PR TITLE
rum_billingFAQ_session replay.md

### PR DESCRIPTION
**Pull request by jamiemilstein**
Added a question clarifying what the usage page shows for Session Replay vs Browser RUM.

### CODE REVIEW

File: content/en/account_management/billing/rum.md [hunk 1]
```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
 ## Does Datadog use cookies?
 
 Yes. Datadog uses cookies to stitch together the various steps of your users into a session. This process does not use cross-domain cookies, and it does not track the actions of your users outside your applications.
+
+## I am seeing Session Replays showing up in my usage page, but I have not configured Session Replay in my environment.
```
Comment (sarina-dd) [inline content/en/account_management/billing/rum.md:34]:
```suggestion
## Session Replays shows up in my usage page, but I have not configured Session Replay in my environment.
```

File: content/en/account_management/billing/rum.md [hunk 2]
```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
...
+
+Session Replay is the name of a SKU. Compared to the Browser SKU, the Session Replay SKU also includes replays, long tasks, and resources. So, if you are collecting any of the three, you will be billed for the Session Replay SKU.
```
Comment (sarina-dd) [inline content/en/account_management/billing/rum.md:36]:
```suggestion
Session Replay includes replays, long tasks, and resources. If you are collecting any of the three, you will be billed for Session Replay.
```

**Review by sarina-dd — COMMENTED** [review]
Left feedback for review

File: content/en/account_management/billing/rum.md (hunk 2 above)
Comment (jamiemilstein) [reply content/en/account_management/billing/rum.md:36]: @MirandaKapin what do you think?

Comment (sarina-dd) [reply content/en/account_management/billing/rum.md:36]: More context to this edit: we do not use the term SKU or contextual billing terminology in the documentation. We should reference as the product name(s).

Comment (jamiemilstein) [reply content/en/account_management/billing/rum.md:36]: understood, that makes sense. thank you for clarifying

File: content/en/account_management/billing/rum.md [hunk 3]
```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
 ## Does Datadog use cookies?
 
 Yes. Datadog uses cookies to stitch together the various steps of your users into a session. This process does not use cross-domain cookies, and it does not track the actions of your users outside your applications.
+
+## Session Replays shows up in my usage page, but I have not configured Session Replay in my environment.
```
Comment (jamiemilstein) [inline content/en/account_management/billing/rum.md:34]: would it be show up instead of show**s** up? @sarina-dd

Comment (sarina-dd) [reply content/en/account_management/billing/rum.md:34]: Good catch. +1 yes, this should be "show up". Also, it should be "Session replays show up", not "Session Replays" as the latter refers to what is physically seen in the UI versus the product entity itself. Will submit an edit!

Comment (sarina-dd) [inline content/en/account_management/billing/rum.md:34]:
```suggestion
## Session replays show up in my usage page, but I have not configured Session Replay in my environment.
```

File: content/en/account_management/billing/rum.md [hunk 4]
```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
 ## Does Datadog use cookies?
 
 Yes. Datadog uses cookies to stitch together the various steps of your users into a session. This process does not use cross-domain cookies, and it does not track the actions of your users outside your applications.
+
+## Session replays show up in my usage page, but I have not configured Session Replay in my environment.
```
Comment (alai97) [inline content/en/account_management/billing/rum.md:34]:
```suggestion
## Session replays show up in my Usage page, but I have not configured Session Replay in my environment
```

File: content/en/account_management/billing/rum.md [hunk 5]
```diff
@@ -30,3 +30,7 @@ Browser logs are automatically tied to the corresponding RUM session, enabling y
...
+
+Session Replay includes replays, long tasks, and resources. If you are collecting any of the three, you will be billed for Session Replay.
```
Comment (alai97) [inline content/en/account_management/billing/rum.md:36]:
```suggestion
Session Replay includes replays, long tasks, and resources. If you are collecting any of the three, you are billed for Session Replay.
```

**Review by alai97 — APPROVED** [review]

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**